Harmonize and fix log messages in HEOS.cpp

MarcBeige opened the pull request:
Let all message start with 'HEOS'. Some log messages start with 'DENON by HEOS', others with 'HEOS by DENON'. The first is incorrect because HEOS is a system from Denon, not the other way around, in both cases Denon should be capitalized, not all caps, and it is not good that there are two styles of messages. Proper labels would be 'HEOS by Denon', 'Denon HEOS' or 'HEOS'. The latter is used as it is simpler and the manufacturer also refers to the system as HEOS.

MarcBeige committed:
Update HEOS.cpp

Let all message start with 'HEOS'. Some log messages start with 'DENON by HEOS', others with 'HEOS by DENON'. The first is incorrect because HEOS is a system from Denon, not the other way around, in both cases Denon should be capitalized, not all caps, and it is not good that there are two styles of messages. Proper labels would be 'HEOS by Denon', 'Denon HEOS' or 'HEOS'. The latter is used as it is simpler and the manufacturer also refers to the system as HEOS.

kiddigital commented:
@MarcBeige , this module still uses an old style of logging using the old logger functions.

If you would remove the '_log.' before each logging/debug statement, it will use the logger in the HardwareBase class that will append the name of the hardware module for you.

So no need anymore to put 'HEOS: ' before each line 😁

MarcBeige committed:
Remove _log. and hardware name from logging

Remove _log. from Log and Debug calls so the hardware name is automatically included in the message. Removed the inconsistent hardware name from the messages.

MarcBeige commented:
I have updated HEOS.cpp to the new style of logging.

gizmocuz commented:
Great, thank you for this!
